code/game/gamemodes/steal_items.dm

```diff
@@ -155,7 +159,33 @@
 /datum/theft_objective/traitor/ablative
 	name = "an ablative armor vest"
 	typepath = /obj/item/clothing/suit/armor/laserproof
-	protected_jobs = list("Head of Security", "Warden")
+	protected_jobs = list("Captain", "Head of Security", "Warden")
+
+/datum/theft_objective/traitor/telebaton
+	name = "a telescopic baton"
+	typepath = /obj/item/weapon/melee/telebaton
+	protected_jobs = list("Captain", "Head of Security")
+
+/datum/theft_objective/traitor/planningframe
+	name = "the law planning frame"
+	typepath = /obj/item/weapon/planning_frame
+	protected_jobs = list("Captain", "Research Director")
+
+/datum/theft_objective/traitor/belt
+	name = "the advanced toolbelt"
+	typepath = /obj/item/weapon/storage/belt/utility/chief
+	protected_jobs = list("Captain", "Chief Engineer")
+
+/datum/theft_objective/traitor/antibody
+	name = "an antibody scanner"
+	typepath = /obj/item/device/antibody_scanner
+	protected_jobs = list("Chief Medical Officer", "Medical Doctor")
+
+/datum/theft_objective/traitor/ttv
+	name = "a tank transfer valve"
+	typepath = /obj/item/device/transfer_valve
+	protected_jobs = list("Research Director", "Scientist")
```

Comment:
I would think its pretty strange.
Also Roboticists are losing their tox access in this PR. Since they have no reason to have it.